dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -134,13 +145,30 @@ private static Options configureOptions() {
     }
   }
 
-  static void configureMavenRepositories(CommandLine commandLine) {
+  static void configureMavenRepositories(CommandLine commandLine) throws ParseException {
     if (!commandLine.hasOption("m")) {
       return;
     }
-    Splitter commaSplitter = Splitter.on(",");
-    for (String mavenRepositoryUrl : commaSplitter.splitToList(commandLine.getOptionValue("m"))) {
-      RepositoryUtility.registerMavenRepository(mavenRepositoryUrl);
+
+    ImmutableList.Builder<RemoteRepository> repositoryListBulder = ImmutableList.builder();
+    for (String mavenRepositoryUrl : commandLine.getOptionValues("m")) {
+      RemoteRepository repository =
+          new RemoteRepository.Builder(mavenRepositoryUrl, "default", mavenRepositoryUrl).build();
```

Comment:
The first parameter is supposed to be the ID, and it can be null. Should we pass null?

Reply:
Set to null.

I had thought setting id with URL would give better error message, but `RemoteRepository.toString` outputs its URL and id. So null is just fine.

```
Could not find artifact org.springframework:nosuchartifact:jar:3.1.0.RC2 in  (https://repo.spring.io/milestone)
```

When null is passed to id, then id is set to an empty string.